src/user/introduction.rst

```diff
@@ -8,8 +8,7 @@ A brief introduction
 
 What is conda-forge?
 --------------------
-
-Conda-forge is a community effort that provides conda packages for a wide range of software.
+Conda-forge can be defined as a compilation of distributions, recipes and build infrastructure that makes the task of installing conda packages hassle-free for a wide range of software. It is a single channel that provides all the up-to-date, compatible versions of conda packages along with all the required dependencies for the conda package manager.
```

Comment:
How about noting that it is open source and driven by the community here? It sort of got taken out by the edit. How does this rough edit sound to you?

```suggestion
The conda-forge project can be viewed as a compilation of distributions, recipes, and build infrastructure that enable installing and distributing a wide range of software as conda packages. In a single channel, the conda-forge community provides up-to-date, compatible versions of conda packages along with all their required dependencies. Our work is driven by the community to provide packaged software free of charge and in a transparent manner.
```